### DISCUSSION TITLE
posts/Export-Import-ConditionalAccessPolicy/

### DISCUSSION THREAD

**Discussion by giscus** [marcmylemans/marcmylemans.github.io#4] [Announcements]
# posts/Export-Import-ConditionalAccessPolicy/

Export and Import Azure Conditional Access Policies

https://mylemans.online/posts/Export-Import-ConditionalAccessPolicy/

**Comment by LukasHanot-Netflow** [comment]
Script searches for the json config files in c:\temp, just a heads up!
